packages/preview/laskutys/1.0.0/docs/color_presets.md

```diff
@@ -0,0 +1,29 @@
+# Color presets
+
+Import presets from the package:
+
+```typst
+#import "@preview/laskutys:1.0.0": PRESET_OCEAN
+```
+
+## PRESET_MINIMAL (default)
+
+![PRESET_MINIMAL](/docs/images/preset_minimal.svg)
```

Comment:
I would recommend using relative paths for these images so they can load both from your own repository and from this one (in which people will end up if they start browsing the docs from Typst Universe).